.tool-versions

```diff
@@ -0,0 +1 @@
+java liberica-8u332+9 zulu-8.62.0.19 corretto-8.332.08.1
```

Comment:
Shouldn't this be pinning to jdk 11?

Reply:
oop, yeah, got merged forward from older versions of this PR

Reply:
I like the idea behind these kinds of tooling, but this provokes me to push a `shell.nix` on my next PR :)